CMakeLists.txt

```diff
@@ -0,0 +1,67 @@
+cmake_minimum_required(VERSION 3.0)
+
+option(HUNTER_ENABLED "Enable Hunter package manager" NO)
+include("cmake/Hunter/HunterGate.cmake")
+HunterGate(
+        URL "https://github.com/ruslo/hunter/archive/v0.19.167.tar.gz"
+        SHA1 "244a13718ea2e6c0c8998d15559c7e459f615708"
+)
+
+project(SDL_ttf VERSION 2.0.14)
+
+hunter_add_package(freetype)
+find_package(freetype CONFIG REQUIRED)
+
+hunter_add_package(SDL2)
+find_package(SDL2 CONFIG REQUIRED)
+
+set(SRC_C_FILES
+        SDL_ttf.c
+        )
+
+set(SRC_H_FILES
+        SDL_ttf.h
+        )
+
+set(SRC_FILES ${SRC_C_FILES} ${SRC_H_FILES})
+
+add_library(${PROJECT_NAME} ${SRC_FILES})
+
+target_link_libraries(${PROJECT_NAME} freetype::freetype SDL2::SDL2 SDL2::SDL2main)
```

Comment:
I guess it should be `target_link_libraries(... PUBLIC ...)`.

Reply:
true, will change that